README.md

```diff
@@ -38,7 +38,10 @@ Ranger: -Pranger-1.0, -Pranger-1.1, -Pranger-1.2 -Pranger-2.0
 
 ### Installation
 
-Place the spark-ranger-&lt;version&gt;.jar into $SPARK_HOME/jars.
+Place the spark-ranger-&lt;version&gt;.jar and dependencies list below into $SPARK_HOME/jars on all nodes.
+
+- gethostname4j-0.0.3.jar
```

Comment:
Any reason to invoke these dependencies?

Reply:
When Spark wants to download policies, it uses RangerRESTUtils class and "com.kstruct.gethostname4j.Hostname" is imported there. In Addition, That class(Hostname.java) uses "com.sun.jna.Platform" class.  It will throw the errors "java.lang.NoClassDefFoundError: com/kstruct/gethostname4j/Hostname" and "java.lang.NoClassDefFoundError: com/sun/jna/Platform"

Example Scenario: Start the Spark Thrift Server(It will be started successfully) and connect to it from the Beeline. It throws those errors in the Spark Thrift Server logs and failed.

Take a look at those classes: 
https://github.com/apache/ranger/blob/master/agents-common/src/main/java/org/apache/ranger/plugin/util/RangerRESTUtils.java
https://github.com/mattsheppard/gethostname4j/blob/master/src/main/java/com/kstruct/gethostname4j/Hostname.java

Reply:
Seems only ranger 2.0 has this issue. Maybe add these to ranger-2.0 profile?

Reply:
I think so. I have not checked other version.

Reply:
So, what will happen to this pull request? You can count on me to contribute.

Reply:
please make the notes more specific. I suggest you create a `docs/installation-addons.md`, put these notes there and use a link here to refer to